tests/datavolume_test.go

```diff
@@ -182,4 +182,50 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]DataVolume tests",
 			}
 		})
 	})
+
+	Describe("Verify DataVolume", func() {
```

Comment:
None of the Describes/Tables/Its mention the block PVC part of the test, and they are specifically designed for the block mode, so if you could add that. Something like

```go
Describe("Verify DataVolume with block mode", func() {
```

Reply:
done